src/sources/fluent/mod.rs

```diff
@@ -236,6 +272,7 @@ impl FluentDecoder {
                             tag: tag.clone(),
                             timestamp,
                             record,
+                            chunk: options.chunk.clone(),
```

Comment:
Can we assume ownership of the chunk field here?

Reply:
That too is inside a `while` loop, so no.